### DISCUSSION TITLE
For Chinese students, how should I fill out my information

### DISCUSSION THREAD

**Discussion by kuankuan2007** [community/community#73430] [GitHub Education]
### Hi everyone,

Github Education always refuses my student certification because of my billing information,
![image](https://github.com/community/community/assets/109161762/8b0ab7f4-7a14-4235-a1fe-887860f6d454)
But the problem is that as a Chinese student, my name is 苟浩铭(Gou Haoming), my last name is 苟(Gou), and my first name is 浩铭(Haoming), both in my student ID card and transcript. However, such last names are too short for Github, and the first and last names are not in the correct order, let alone recognized correctly

**Comment by Seol-JY** [comment]
Does the same problem occur even if I use an English name?

**Reply by kuankuan2007** [reply]
If the student ID card and billing information match, it is OK. The problem is that my student ID card is written in Chinese

**Reply by Seol-JY** [reply]
@kuankuan2007 
It would be good if you have an international student ID or proof of attendance in English, but if you don't have one, you will need help from Github. I'm sorry I couldn't help you.

**Reply by kuankuan2007** [reply]
That's right

**Comment by Jerry12228** [comment]
Hi I have a same question. Did you solve that? Could you help me?

**Reply by llj123456789** [reply]
me，too

**Reply by llj123456789** [reply]
help ，help

**Reply by kuankuan2007** [reply]
I'm sorry. I didn't solve it either

**Comment by kiterza** [comment]
The GitHub education team is really stupid. The automatic detection program does not have the ability to detect Chinese materials, nor can it accept manual processing. Even if the Chinese education department provides an API for verifying student identities, and even jetbrains already supports it, GitHub education still No, this problem has been around for at least two years and no one has solved it.

**Reply by kiterza** [reply]
It’s difficult to contact the GitHub education team. The first time I passed the certification, I even contacted Microsoft. I really don’t know how such a closed team can provide services.

**Comment by vitojan** [comment]
我在姓氏後面添加一個空格，目前測試可以通過自動檢測。
![image](https://github.com/community/community/assets/56464847/e35f81d6-f3b5-47b1-b2a5-20e52faa6ccf)

**Reply by HCOOH46** [reply]
Thanks! The problem is solved!

**Reply by EtMemor** [reply]
You 're a genius! Thanks

**Reply by ixijxjgxidj-cmd** [reply]
天才

**Comment by llj123456789** [comment]
你好，已经收到你的邮件信息

**Comment by llj123456789** [comment]
你好，已经收到你的邮件信息

**Comment by llj123456789** [comment]
你好，已经收到你的邮件信息
